docs/conf.py

```diff
@@ -25,7 +25,7 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-autodoc_mock_imports = ["fontio"]
+autodoc_mock_imports = ["displayio", "fontio"]
```

Comment:
Was this required to get it to pass CI?

Reply:
Correct, I added this to pass CI.

Reply:
Got it, the trick is actually to add the Blinka version of `displayio` as a dependency now that we're using it.  I'll suggest those changes in another review so it's clear!

Reply:
You can add the following package to the main `requirements.txt` file in the main folder, as well as `setup.py` in the `install_requires` list:

requirements.txt
```python
# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
#
# SPDX-License-Identifier: Unlicense

Adafruit-Blinka
Adafruit-Blinka-displayio

```

---
setup.py
```python
    ...
    author="Adafruit Industries",
    author_email="circuitpython@adafruit.com",
    install_requires=[
        "Adafruit-Blinka",
        "Adafruit-Blinka-displayio",
    ],
    # Choose your license
    license="MIT",
    ...
```